$MirOS: ports/audio/faad/patches/patch-common_mp4ff_mp4ff_int_types_h,v 1.1.7.1 2005/03/18 15:42:52 tg Exp $
--- common/mp4ff/mp4ff_int_types.h.orig	2004-08-11 00:31:01.000000000 +0200
+++ common/mp4ff/mp4ff_int_types.h	2004-08-11 00:31:29.000000000 +0200
@@ -15,6 +15,6 @@ typedef unsigned __int64 uint64_t;
 
 #else
 
-#include <stdint.h>
+#include <inttypes.h>
 
 #endif
